src/main/java/kitchenpos/order/domain/Order.java

```diff
@@ -0,0 +1,85 @@
+package kitchenpos.order.domain;
+
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.table.domain.OrderTable;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@EntityListeners(AuditingEntityListener.class)
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "order_table_id", foreignKey = @ForeignKey(name = "fk_orders_order_table"))
+    private OrderTable orderTable;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private OrderStatus orderStatus;
+
+    @CreatedDate
+    @Column(nullable = false)
+    private LocalDateTime orderedTime;
+
+    @Embedded
+    private OrderLineItems orderLineItems = new OrderLineItems();
+
+    protected Order() {
+    }
+
+    public Order(OrderTable orderTable) {
+        validateOrderTable(orderTable);
+        this.orderTable = orderTable;
+        orderStatus = OrderStatus.COOKING;
```

Comment:
상태 변경은 한 곳에서 해주도록 하는 것도 좋을 것 같아요 :)

```suggestion
        changeOrderStatus(OrderStatus.COOKING)
```